examples/third_party_integration/envtest/README.md

```diff
@@ -0,0 +1,25 @@
+# EnvTest Integration
+
+`envtest` is a companion project to be used with `kubebuilder` based projects where you are creating your own controllers based on CRDs to enable quick test workflows.
```

Comment:
Although this does something similar, in functionality,  as envtest, make sure to mention that this is not envtest and does not use it.